contracts/contracts/token/OUSD.sol

```diff
@@ -174,22 +192,23 @@ contract OUSD is Initializable, InitializableERC20Detailed, Governable {
         );
     }
 
+    // Backwards compatible view
+    function nonRebasingCreditsPerToken(address _account)
+        external
+        view
+        returns (uint256)
+    {
+        return alternativeCreditsPerToken[_account];
+    }
+
     /**
      * @dev Transfer tokens to a specified address.
      * @param _to the address to transfer to.
      * @param _value the amount to be transferred.
      * @return true on success.
      */
-    function transfer(address _to, uint256 _value)
-        public
-        override
-        returns (bool)
-    {
+    function transfer(address _to, uint256 _value) public returns (bool) {
```

Comment:
nit: this and other `public` methods could be `external`

Reply:
agreed